<launch>	
  
  <arg name="robot" default="coman"/>
<remap from="/joint_states" to="/footstep/joint_states"/>

	<node pkg="footstep_planner" name="foot_planner" type="foot_planner" respawn="false" output="screen" cwd="node" args="$(arg robot)">
		<!--topic remapping-->
		<!-- <remap from="input_cloud" to="/camera/depth_registered/points" /> -->
		<param name="voxel_size" value="0.01" />
		<param name="normal_radius" value="0.08" />
		<param name="curvature_threshold" value="0.04" />
		<param name="min_cluster_size" value="50" /> <!-- with 0.01 in voxel size, 50 seems reasonable to place a foot -->
		<param name="cluster_tolerance" value="0.05" />
		<param name="feasible_area" value="2.5" />
	</node>
</launch>
